guides/auth-torii-with-github.md

```diff
@@ -634,27 +634,32 @@ import config from '../config/environment';
 
 export default class ToriiAuthenticator extends Torii {
   @service torii;
-  @service ajax;
 
   async authenticate() {
-    const tokenExchangeUri = config.torii.providers['github-oauth2'].tokenExchangeUri;
+    const tokenExchangeUri = config.torii.providers['google-oauth2'].tokenExchangeUri;
 
     let data = await super.authenticate(...arguments);
-    let response = await this.ajax.request(tokenExchangeUri, {
-      type: 'POST',
-      crossDomain: true,
-      dataType: 'json',
-      contentType: 'application/json',
-      data: JSON.stringify({
+    const response = await fetch(tokenExchangeUri, {
+      // Adding method type 
+      method: "POST",
+      // no-cors, *cors, same-origin
+      mode: 'cors',
+      // Adding body or contents to send 
+      body: JSON.stringify({
         authorizationCode: data.authorizationCode
+      }),
+      // Adding headers to the request 
+      headers: {
+        'Content-Type': 'application/json' // 'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
```suggestion
        'Content-Type': 'application/json'
```

Or is the comment relevant?

Reply:
Thought it might be a good idea to have it at the time.